v1.1/uuid.md

```diff
@@ -0,0 +1,135 @@
+---
+title: UUID
+summary: The UUID data type stores 128-bit Universal Unique Identifiers.
+toc: false
+---
+
+<span class="version-tag">New in v1.1:</span> The `UUID` (Universally Unique Identifier) [data type](data-types.html) stores a 128-bit value that is [unique across both space and time](https://www.ietf.org/rfc/rfc4122.txt). 
+
+<div id="toc"></div>
+
+## Syntax
+A `UUID` value can be expressed using the following formats:
+
+Format | Description 
+-------|-------------
+Standard [RCF4122](http://www.ietf.org/rfc/rfc4122.txt)-specified format | Hyphen-seperated groups of 8, 4, 4, 4, 12 hexadecimal digits.<br><br> Example: acde070d-8c4c-4f0d-9d8a-162843c10333
```

Comment:
Let's put backticks around the example values in each row, e.g.:

> Example: `acde070d-8c4c-4f0d-9d8a-162843c10333`

Sorry for not catching this earlier.